tests/mcp_server.py

```diff
@@ -202,6 +202,22 @@ async def use_sampling(ctx: Context, foo: str) -> str:  # type: ignore
     return result.model_dump_json(indent=2)
 
 
+class UserResponse(BaseModel):
+    response: str
+
+
+@mcp.tool()
+async def use_elicitation(ctx: Context[ServerSessionT, LifespanContextT, RequestT], question: str) -> str:
```

Comment:
This is not the correct type.

It's probably this:
```suggestion
async def use_elicitation(ctx: Context[ServerSession, None], question: str) -> str:
```

Reply:
i noticed the other mcp test functions (`echo_deps`, `use_sampling`) all use `Context[ServerSessionT, LifespanContextT, RequestT]` as types. for consistency, should we update other functions or is there a reason `use_elicitation` should use concrete types instead of generics?